docs/content/latest/explore/ysql-language-features/partitions.md

```diff
@@ -95,49 +102,51 @@ Each month range in the preceding examples includes the start of the month, but
 Optionally, you can create an index on the key columns and other indexes for every partition, as follows:
 
 ```sql
-CREATE INDEX ON employees_2019_02 (change_date);
-CREATE INDEX ON employees_2019_03 (change_date);
+CREATE INDEX ON order_changes_2019_02 (change_date);
+CREATE INDEX ON order_changes_2019_03 (change_date);
 ...
-CREATE INDEX ON employees_2020_11 (change_date);
-CREATE INDEX ON employees_2020_12 (change_date);
-CREATE INDEX ON employees_2021_01 (change_date);
+CREATE INDEX ON order_changes_2020_11 (change_date);
+CREATE INDEX ON order_changes_2020_12 (change_date);
+CREATE INDEX ON order_changes_2021_01 (change_date);
 ```
 
 Partitioning is a flexible technique that allows you to remove old partitions and add new partitions for  new data when required. You do this by changing the partition structure instead of the actual data.
 
 The following example shows how to remove the partition from the partitioned table while retaining access to it as a separate table which enables you to perform operations on the data:
 
 ```sql
-ALTER TABLE employees DETACH PARTITION employees_2019_03;
+ALTER TABLE order_changes DETACH PARTITION order_changes_2019_03;
 ```
 
 The following example shows how to add a new partition to deal with new data by creating an empty partition in the partitioned table:
 
 ```sql
-CREATE TABLE employees_2021_02 PARTITION OF employees
+CREATE TABLE order_changes_2021_02 PARTITION OF order_changes
   FOR VALUES FROM ('2021-02-01') TO ('2021-03-01');
 ```
 
 Note the following:
 
+- The primary key for a table should always contain partition key. In the current release of YugabyteDB, it is recommended to set the primary key directly on partitions.
```

Comment:
Nit: The primary key for a partitioned table should always contain the primary key.

Reply:
@deeps1991 I'll add "partitioned" before "table". However, I don't understand "The primary key ... should contain the primary key". Did you mean "The primary key for a partitioned table should always contain the partition key" ?

Reply:
We now have: 
The primary key for a partitioned table should always contain the partition key.
I'm going to merge this PR because (1) Deepthi approved it already, (2) she's currently on vacation, (3) the change is very minor anyway, and (4) we need the updated document for 2.7 release